play/play-v28/RotatingSecretComponents.scala

```diff
@@ -21,6 +25,9 @@ trait RotatingSecretComponents extends BuiltInComponentsFromContext {
 
   override lazy val requestFactory: RequestFactory =
     RotatingSecretComponents.requestFactoryFor(secretStateSupplier, httpConfiguration)
+
+  override lazy val csrfTokenSigner: CSRFTokenSigner =
+    new RotatingSecretComponents.RotatingKeyCSRFTokenSigner(secretStateSupplier, systemUTC())
```

Comment:
This the essential bit of this fix - although `RotatingSecretComponents` has always overridden the `RequestFactory` to handle rotating the [Play Application Secret](https://www.playframework.com/documentation/3.0.x/ApplicationSecret), we forgot to override the `CSRFTokenSigner` to teach _it_ about rotating secrets.

* `RequestFactory` - includes components that handle signing of [Session & Flash cookies](https://www.playframework.com/documentation/3.0.x/SettingsSession)
* `CSRFTokenSigner` - signs the CSRF token used for [CSRF checking](https://www.playframework.com/documentation/3.0.x/ScalaCsrf)